.github/workflows/migrations.yaml

```
@@ -0,0 +1,165 @@
+      svc: ${{ matrix.service }}
+      POSTGRES_HOST: localhost
+      POSTGRES_PORT: 5432
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_DB: postgres # will be overwritten in "Pull staging data"
+
+    services:
+      postgres:
+        image: postgres
+        # Set health checks to wait until postgres has started
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        env:
+          POSTGRES_HOST: localhost
+          POSTGRES_PORT: 5432
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: postgres
+        ports:
+          - 5432:5432
+    steps:
+      - name: Setup flyctl
+        uses: superfly/flyctl-actions/setup-flyctl@master
+        with:
+          version: 0.1.66
+      # currently ubuntu-latest ships with psql 14.x,
+      # whose pg_dump is incompatible with our postgres server (15.x)
+      - name: Setup postgres client
+        run: |
+          sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list'
+          wget -qO- https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo tee /etc/apt/trusted.gpg.d/pgdg.asc &>/dev/null
+          sudo apt update
+          sudo apt remove postgresql-client
+          sudo apt install postgresql-client-15 -y
+          /usr/lib/postgresql/15/bin/pg_dump --version
+      - name: Clone Repo
+        uses: actions/checkout@v3
+      - name: Open tunnel to staging db
+        run: |
+          flyctl proxy 5431:5432 -a helpwave-staging-postgres &
+          sleep 5
+        env:
+          FLY_API_TOKEN: ${{ secrets.STAGING_DB_TOKEN }}
+      - name: Pull staging data
+        run: |
+          # service-name -> service_name
+          REMOTE_DB=$(echo "$REMOTE_DB" | sed "s/-/_/g")
+          # Build URIs
+          REMOTE=postgres://$STAGING_DB_USER:$STAGING_DB_PASS@localhost:5431/$REMOTE_DB
+          LOCAL=postgres://$POSTGRES_USER:$POSTGRES_PASSWORD@$POSTGRES_HOST:$POSTGRES_PORT/$POSTGRES_DB
+          # Remote -> SQL -> Local
+          /usr/lib/postgresql/15/bin/pg_dump -C -c --if-exists --no-comments -O -x $REMOTE -f dump.sql
+          /usr/lib/postgresql/15/bin/psql -q $LOCAL -f dump.sql
+          # Use newly created database from now on
+          echo "POSTGRES_DB=$REMOTE_DB" >> "$GITHUB_ENV"
+        env:
+          REMOTE_DB: helpwave_staging_${{ matrix.service }}
+          STAGING_DB_USER: ${{ secrets.STAGING_DB_USER }}
+          STAGING_DB_PASS: ${{ secrets.STAGING_DB_PASS }}
+      - name: Collect current version
+        id: collect-version
+        run: |
+          echo -n "VERSION=" > $GITHUB_OUTPUT
+          # for some reason, beyond my comprehension,
+          # the output of migrate is sent to stderr in the CI, but stdout on local
+          ./migrate.sh task-svc version 2>&1 | tail -n1 >> $GITHUB_OUTPUT
```

```suggestion
          ./migrate.sh task-svc version 2>&1 | tail -n1 >> $GITHUB_OUTPUT
```
`task-svc` should be the current service